packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -594,44 +604,46 @@ function DashboardLayout(props: DashboardLayoutProps) {
           </Stack>
         </Toolbar>
       </AppBar>
-      <Drawer
-        container={appWindow?.document.body}
-        variant="temporary"
-        open={isMobileNavigationExpanded}
-        onClose={handleSetNavigationExpanded(false)}
-        ModalProps={{
-          keepMounted: true, // Better open performance on mobile.
-        }}
-        sx={{
-          display: {
-            xs: 'block',
-            sm: disableCollapsibleSidebar ? 'block' : 'none',
-            md: 'none',
-          },
-          ...getDrawerSharedSx(false),
-        }}
-      >
-        {getDrawerContent(false, 'Phone')}
-      </Drawer>
-      <Drawer
-        variant="permanent"
-        sx={{
-          display: {
-            xs: 'none',
-            sm: disableCollapsibleSidebar ? 'none' : 'block',
-            md: 'none',
-          },
-          ...getDrawerSharedSx(isMobileMini),
-        }}
-      >
-        {getDrawerContent(isMobileMini, 'Tablet')}
-      </Drawer>
-      <Drawer
-        variant="permanent"
-        sx={{ display: { xs: 'none', md: 'block' }, ...getDrawerSharedSx(isDesktopMini) }}
-      >
-        {getDrawerContent(isDesktopMini, 'Desktop')}
-      </Drawer>
+      {!hideNavigation && <React.Fragment>
```

Comment:
`React.Fragment` not needed?

Reply:
Also can we write it like `{!hideNavigation ? ... : null}`? Just because we've been following that convention in the project.